test/system/120-load.bats

```diff
@@ -57,9 +57,14 @@ verify_iid_and_name() {
     # Load using -i; this time the image should be tagged.
     run_podman load -i $archive
     verify_iid_and_name $img_name
+    run_podman rmi $iid
 
-    # Same as above, using stdin
+    # Also make sure that `image load` behaves the same.
+    run_podman image load -i $archive
```

Comment:
LGTM but wouldn't a better test be to run `image load < $archive` ?

Reply:
I wanted to use the reproducer from the issue. That's effectively the same code path afaics.